docker/ingestion/README.md

```diff
@@ -3,18 +3,13 @@
 Refer to [DataHub Metadata Ingestion](../../metadata-ingestion/mce-cli) to have a quick understanding of the architecture and 
 responsibility of this service for the DataHub.
 
-## Build
+## Build & Run
 ```
- docker build -t ingestion -f docker/ingestion/Dockerfile .
+cd docker/ingestion && docker-compose up --build
 ```
-This command will build and deploy the image in your local store.
+This command will rebuild the docker image and start a container based on the image.
 
-## Run container
-```
- cd docker/ingestion && docker-compose up
-```
-This command will start the container. If you have the image available in your local store, this image will be used
-for the container otherwise it will build the image from local repository and then start that.
+To start a container using an existing image, run the same command without the `--build` flag.
```

Comment:
Same comment as in the other PR: https://github.com/linkedin/datahub/pull/1687

Reply:
We never publish this image so will need to always build it

Reply:
Oh haha correct :)